src/Gameplay/Deck.java

```diff
@@ -0,0 +1,40 @@
+package Gameplay;
+
+import Cards.Card;
+import Cards.Suit;
+
+import java.awt.*;
+import java.io.IOException;
+import java.util.Collections;
+
+public class Deck extends Pile {
+
+    public Deck(int x, int y) throws IOException {
```

Comment:
Nothing throws `IOException` in here

```suggestion
    public Deck(int x, int y) {
```